### DISCUSSION TITLE
KNX telegram are not sent

### DISCUSSION THREAD

**Discussion by platini76** [ascillato/Tasmota_KNX#590] [Q&A]
hello, thanks for your work..
I installed a tasmota KNX self compiled on my esp32
this is configuration:

<img width="742" alt="Screenshot 2022-10-02 alle 15 18 10" src="https://user-images.githubusercontent.com/10352902/193456132-622ef26d-6782-4619-bc22-e33d72cdefc9.png">

this is also my light fixture configuration on my thinknx server:

<img width="599" alt="Screenshot 2022-10-02 alle 15 18 57" src="https://user-images.githubusercontent.com/10352902/193456172-97733896-deb5-4aef-a05f-b314ef4caedd.png">

no telegram are sent or received...

what can be the problem?

**Comment by platini76** [comment]
<img width="624" alt="Screenshot 2022-10-03 alle 02 39 02" src="https://user-images.githubusercontent.com/10352902/193483851-05ce2255-ebbc-474b-a1c4-487cd75093c4.png">
sorry I discover a little error.. but telegram are not sent

**Comment by platini76** [comment]
telegram are sent but not to the server...  which problem I have to solve?
<img width="915" alt="Screenshot 2022-10-03 alle 02 40 30" src="https://user-images.githubusercontent.com/10352902/193483931-5bbad29a-9426-42b0-b4e8-af6811befdca.png">

**Comment by platini76** [comment]
noone can help me?

**Reply by barbudor** [reply]
@platini76 I know nothing about KNX, sorry
My help yesterday as only about the problem of saving from the web page

From what I read on this repository, this was a temporary fork of Tasmota before the specific KNX feature was integrated in Tasmota

So I strongly suggest that you ask all your questions on Tasmota Discussions

**Comment by Noschvie** [comment]
Do you have IP multicast enabled at your wifi / WLAN setup ?
Maybe you should use Wireshark to check the traffic over the network.

**Comment by platini76** [comment]
it is possible that my knx router (thinknx) did not work properly?
any suggestion on a simple software router that can be installed also on a mac?
